pkg/runtime/ast/show.go

```diff
@@ -527,3 +527,16 @@ func (s *ShowReplicaStatus) Restore(flag RestoreFlag, sb *strings.Builder, args
 	sb.WriteString("SHOW REPLICA STATUS")
 	return s.baseShow.Restore(flag, sb, args)
 }
+
+type ShowUsers struct {
+	*baseShow
```

Comment:
Parameter of Tenant should be defined，Such as：
      SHOW USER FROM `arana`
Parameter of `arana` should be parse.

Reply:
By the way, UT and test/integration_test.go should be added  :)